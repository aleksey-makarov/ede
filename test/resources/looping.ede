{
  "numbers": [1, 2, 3],
  "strings": ["foo", "bar"],
  "indexed": [false, true, false, false],
  "keyed": {
    "a": 1.0,
    "b": 2.0,
    "c": 3.0
  },
  "empty": []
}
---
{% for n in numbers %}
  {{ n.value }}
{% endfor %}

{% for var in strings %}
  {{ var.value }}
{% endfor %}

{% for var in indexed %}
  {{ var.index }}:{{ var.value }}
{% endfor %}

{% for var in keyed %}
  {{ var.key }}:{{ var.value }}
{% endfor %}

{% for var in empty %}
  Shit!
{% else %}
  empty
{% endfor %}
