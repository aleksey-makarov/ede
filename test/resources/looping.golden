  1
  2
  3

  foo
  bar

  1:false
  2:true
  3:false
  4:false

  a:1.0
  b:2.0
  c:3.0

  empty
